components/Cell.js

```diff
@@ -125,20 +128,20 @@ const Cell = props => {
       case 'Detail':
         return (
           <View style={_styles.cellAccessoryView}>
-            <View style={_styles.accessory_detail}>
+            <TouchableOpacity style={_styles.accessory_detail} onPress={onPressDetail} activeOpacity={0.7} disabled={isDisabled}>
               <Text style={_styles.accessory_detailText}>i</Text>
-            </View>
+            </TouchableOpacity>
           </View>
         );
       case 'DetailDisclosure':
         return (
           <View style={_styles.cellAccessoryView}>
-            <View style={_styles.accessory_detailDisclosure}>
+            <TouchableOpacity style={_styles.accessory_detailDisclosure} onPress={onPressDetailDisclosure} activeOpacity={0.7} disabled={isDisabled}>
```

Comment:
1. Shouldn't `<View style={_styles.accessory_disclosureIndicator} />` be excluded from `onPressDetailDisclosure`?
2. Does the nesting of TouchableHighlight and TouchableOpacity work fine?

Reply:
1. Sorry I don't get the point here, are you suggesting to move the TouchableOpacity to its' first children?

2. Yes it works fine. I am using this in prod with RN 58.6

Reply:
1. `onPressDetailDisclosure` should only get triggered by pressing the "i". Button Pressing the arrow "disclosureIndicator" should still trigger regular `onPress`. Right?
> Rename `onPressDetailDisclosure` to `onPressDetailDisclosureButton` (https://developer.apple.com/design/human-interface-guidelines/ios/controls/buttons/)
> Move line 143 outside of `TouchableOpacity`